specs/assets_overview/02_user_interface.rst

```diff
@@ -0,0 +1,58 @@
++-------+----------------------+-----------+------------+
+| issue | title                | author    | created    |
++-------+----------------------+-----------+------------+
+| 71_   | Transaction overview | tschubotz | 2019-01-22 |
++-------+----------------------+-----------+------------+
+
+.. _71: https://github.com/gnosis/safe/issues/71
+
+User interface
+==============
+
+.. contents:: Table of Contents
+
+1. Transation list
+------------------
+
+Sections:
++++++++++
+
+- Pending: All pending transactions
+- Today: All transactions of today
+- Yesterday: All transactions of yesterday
+- Please put all older transactions in a section that uses the date in the section title.
+    - Date format: Please use the selected locale if possible.
+
+Types:
+++++++
+- Incoming ETH
+- Incoming ERC20 token
+- Outgoing ETH
+- Outgoing ERC20 token
+- Data
+- Settings change
+
+State:
+++++++
+- Pending
+- Successful
+- Failed
+
+
+2. Transction details - Outgoing ETH
+------------------------------------
+
+2.1. Transction details - Failed
+--------------------------------
+
+2.2. Transction details - Pending
+---------------------------------
+
+2.3. Transction details - Outgoing ETH
```

Comment:
this is a little confusing because the title of `2` is the same

Reply:
yeah, it should say "token" 😇